althkmabod. py

```diff
@@ -0,0 +1,582 @@
+
+import zipfile
+import os
+try:
+    from telethon.sessions import StringSession
+    import asyncio, re, json, shutil
+    from kvsqlite.sync import Client as uu
+    from telethon.tl.types import KeyboardButtonUrl
+    from telethon.tl.types import KeyboardButton, ReplyInlineMarkup
+    from telethon import TelegramClient, events, functions, types, Button
+    from telethon.tl.types import DocumentAttributeFilename
+    from plugins.converter import MangSession
+    import time, datetime, random 
+    from datetime import timedelta
+    from telethon.errors import (
+        ApiIdInvalidError,
+        PhoneNumberInvalidError,
+        PhoneCodeInvalidError,
+        PhoneCodeExpiredError,
+        SessionPasswordNeededError,
+        PasswordHashInvalidError
+    )
+    from plugins import *
+    from plugins.messages import *
+    from plugins.get_gift import *
+except:
+    os.system("pip install telethon kvsqlite")
+    try:
+        from telethon.sessions import StringSession
+        import asyncio, re, json, shutil
+        from kvsqlite.sync import Client as uu
+        from telethon.tl.types import KeyboardButtonUrl
+        from telethon.tl.types import KeyboardButton
+        from telethon import TelegramClient, events, functions, types, Button
+        from telethon.tl.types import DocumentAttributeFilename
+        from plugins.converter import MangSession
+        import time, datetime, random 
+        from datetime import timedelta
+        from telethon.errors import (
+            ApiIdInvalidError,
+            PhoneNumberInvalidError,
+            PhoneCodeInvalidError,
+            PhoneCodeExpiredError,
+            SessionPasswordNeededError,
+            PasswordHashInvalidError
+        )
+        from plugins import *
+        from plugins.messages import *
+        from plugins.get_gift import *
+    except Exception as errors:
+        print('An Erorr with: ' + str(errors))
+        
+        exit(0)
+
+        
+if not os.path.isdir('database'):
+    os.mkdir('database')
+
+API_ID = "829746"
+API_HASH = "98483e947d2d31c28605e52d368b445f"
+admin ="6569663552"
+
+# Replace with your bot token
+token = "7124198346:AAEGLUcvqbHs-KS2H0zWxSmj7z3Up568obE"
```

Comment:
You probably want to rotate these tokens, since they've now been leaked publicly.